proto/stream_plan.proto

```diff
@@ -107,9 +107,10 @@ message SourceNode {
     TABLE = 1;
     SOURCE = 2;
   }
-  uint32 table_id = 1;
+  uint32 table_id = 1; // use table_id / source_id to fetch SourceDesc from local source manager
```

Comment:
Is it possible keep using `table_id` to access `state_store`, and rename now `table_id` to other in order to keep the concept with other operators